utilities/src/Types/Event.res

```diff
@@ -0,0 +1,76 @@
+/** JSDoc */
+type exceptionType<'a> = {
+    values?: array<Exception.exception_<'a>>,
+  }
+type transactionInfo = {
+    soruce: Transaction.transactionSource
+}
+type thread<'a> = {
+    values: array<Thread.thread<'a>>
+}
+
+type captureContext //TODO
+
+/**
+ * The type of an `Event`.
+ * Note that `ErrorEvent`s do not have a type (hence its undefined),
+ * while all other events are required to have one.
+ */
+type eventType = [ #transaction | #profile | #replay_event | #feedback ]
+type levelType = Severity(Severity.severity) | SeverityLevel(Severity.severityLevel)
+type event<'a>= {
+  event_id?: string,
+  message?: string,
+  timestamp?: float,
+  start_timestamp?: float,
+  // eslint-disable-next-line deprecation/deprecation
+  level?: levelType,
```

Comment:
Here level can just Severity.severityLevel since if you see in Severity file Severity.severity is deprecated
So no need of variants in line 20

Reply:
I saw this...but its better if it throws warning saying its deprecated right, since this type has not been completely removed